tracking/tracking.go

```diff
@@ -24,22 +27,25 @@ func LoadRancherUUID() string {
 	})
 
 	if err != nil {
-		log.Warnf("Error creating client: %v", err)
-		return nilUUID
+		logger.WithField("error", err.Error()).Fatal("Failed to create client")
```

Comment:
logging at fatal level anywhere other than at the highest possible level in the code is a bad pattern. return an error and let it bubble up until it cannot bubble up anymore